### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by ptomato**
Miscellaneous fixes.

### CODE REVIEW

File: spec/abstractops.html [hunk 1]
```diff
@@ -662,9 +662,9 @@ <h1>ParseTemporalTimeZoneString ( _isoString_ )</h1>
...
         1. Return the new Record: {
-          [[OffsetSign]]: *undefined*,
-          [[OffsetHour]]: *undefined*,
-          [[OffsetMinute]]: *undefined*,
+          [[OffsetSign]]: 1,
+          [[OffsetHour]]: +0,
+          [[OffsetMinute]]: +0,
           [[Name]]: `"UTC"`
           }.
       1. If _hour_ is not *undefined*, then
```
Comment (Ms2ger) [inline spec/abstractops.html:671]: Can you clarify the invariant you're aiming for? `ParseTemporalTimeZone` expects that some of the values can be undefined, still (though the result is ignored). Not sure what `ParseTemporalAbsolute` expects.

File: polyfill/lib/timezone.mjs [hunk 2]
```diff
@@ -175,6 +175,7 @@ export class TimeZone {
         GetSlot(dateTime, MICROSECOND),
         GetSlot(dateTime, NANOSECOND)
       );
+      if (epochNs === null) throw new RangeError('DateTime outside of supported range');
```
Comment (Ms2ger) [inline polyfill/lib/timezone.mjs:178]: This looks like it deserves a test.

**Review by Ms2ger — CHANGES_REQUESTED** [review]

**Review by ryzokuken — APPROVED** [review]
LGTM with @Ms2ger's comments addressed.

File: spec/abstractops.html (hunk 1 above)
Comment (ptomato) [reply spec/abstractops.html:671]: I have updated the callers and attempted to describe the invariant better in the commit message.

**Comment by ptomato** [comment]
@Ms2ger Can you check this one again? It can't be merged until you mark that the requested changes have been addressed.

**Comment by ptomato** [comment]
Oh, never mind, you can "dismiss review" if the comments have been addressed ... which strikes me as a bit rude though!

**Pull request merged**